zerver/tests/mails/simple.txt

```diff
@@ -0,0 +1,7 @@
+To: {stream_to_address}
+From: {sender}
+Subject: Testing Email Mirror
+Content-Type: text/plain; charset=utf-8; format=flowed
+Content-Transfer-Encoding: 7bit
+
+This is a plain-text message for testing Zulip.
```

Comment:
this should go under e.g. `zerver/fixtures/email` (since `zerver/fixtures` is where we put this sort of test data in generate